nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -705,11 +707,15 @@ in
     };
 
     systemd.services.nginx-config-reload = mkIf cfg.enableReload {
-      wantedBy = [ "nginx.service" ];
+      wants = [ "nginx.service" ];
+      wantedBy = [ "multi-user.target" ];
       restartTriggers = [ configFile ];
+      stopIfChanged = false;
+      serviceConfig.Type = "oneshot";
       script = ''
         if ${pkgs.systemd}/bin/systemctl -q is-active nginx.service ; then
-          ${pkgs.systemd}/bin/systemctl reload nginx.service
+          ${execCommand} -s reload && \
```

Comment:
Is the reload command following this still necessary if this command already performs a reload?